File: C/Python/114.py
N = int(input())
ls = [input() for i in range(N)]
for i in range(N-1):
    if ls[i][-1] == ls[i+1][0]:
        continue
    else:
        print(ls[i][-1], ls[i+1][0])
        exit()
        
print("Yes")

File: C/JavaScript/114.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split("\n");

const N = parseInt(input[0], 10);
const ls = input.slice(1, N+1);

for (let i = 0; i<N-1; i++){
    const lastChar = ls[i][ls[i].length - 1];
    const firstChar = ls[i+1][0];

    if (lastChar == firstChar){
        continue;
    }else{
        console.log(lastChar, firstChar);
        process.exit(0);
    }
}

console.log("Yes")